src/encoded/static/components/rnaget.js

```diff
@@ -9,6 +9,43 @@ import QueryString from '../libs/query_string';
 import * as globals from './globals';
 import { TableItemCount } from './objectutils';
 import { FacetList, TextFilter } from './search';
+import { FetchedData, Param } from './fetched';
+
+
+const AutocompleteBox = (props) => {
+    const terms = props.suggestions['@graph'];
+    const { handleClick } = props;
+
+    /* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/no-noninteractive-tabindex, jsx-a11y/click-events-have-key-events, arrow-body-style */
+
+    if (terms && terms.length > 0) {
```

Comment:
This is not the style we normally use. We normally use  something like- 
x == 1 ? y : z

Reply:
oh true, I will push a change